src/mon/ElectionLogic.h

```diff
@@ -153,6 +153,13 @@ class ElectionLogic {
    */
   int leader_acked;
 public:
+  enum election_strategy {
```

Comment:
nit, how about using scoped enum?

Reply:
Hmm I don't think I've run across these before; I'll read up on them. In a quick glance it doesn't seem to like the differences matter here though.